package.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "tree-sync",
-  "version": "1.3.0",
+  "version": "1.3.1",
```

Comment:
mind rolling back this 1 line? In general, I prefer to leave the version bump to the time of release. This ensures the version bump is atomic with the release bump.

Reply:
Fixed

Reply:
thank you!